build/ci/job-template.yml

```diff
@@ -49,6 +49,7 @@ jobs:
       - script: brew update && brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/f5b1ac99a7fba27c19cee0bc4f036775c889b359/Formula/libomp.rb && brew unlink python@2 && brew install mono-libgdiplus gettext && brew link gettext --force && brew link libomp --force
         displayName: Install build dependencies
     - ${{ if and( eq(parameters.nightlyBuild, 'true'), eq(parameters.name, 'Ubuntu_x64_NetCoreApp21')) }}:
+      - script: sudo locale-gen en_US.UTF-8 && sudo update-locale
```

Comment:
Will the sudo commands succeed on the server? I am curious to find out.

Reply:
Seems not:

/__w/_temp/1707e74d-1295-4ff9-815a-b62ba21f6a2a.sh: line 1: sudo: command not found


---
In reply to: [379053227](https://github.com/dotnet/machinelearning/pull/4834#discussion_r379053227) [](ancestors = 379053227)